### PR TITLE
Documented different lock status

### PR DESCRIPTION
**Pull request by hackaugusto**

### CODE REVIEW

File: raiden/transfer/state.py [hunk 1]
```diff
@@ -1368,8 +1368,15 @@ def __init__(self, address: Address, balance: Balance):
         self.address = address
         self.contract_balance = balance
 
+        #: Locks which have been introduced with a locked transfer, however the
```
Comment (palango) [inline raiden/transfer/state.py]: What is the colon for?

Comment (hackaugusto) [reply raiden/transfer/state.py]: It's for documentation using sphinx.

https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html#example-google

Comment (palango) [reply raiden/transfer/state.py]: Ah nice, didn't know that. But then you forgot to add them to the comment of `secrethashes_to_onchain_unlockedlocks`

Comment (hackaugusto) [reply raiden/transfer/state.py]: done

**Pull request merged**